projects/sandbox/generate_background/generate_background.py

```diff
@@ -53,7 +53,6 @@ def main(
             "Background data already exists"
             " and forced generation is off. Not generating background"
         )
-        return
```

Comment:
What was the problem here? Don't think we wan't to remove these.

Reply:
Yeah, I thought it might be the most straightforward fix, but now I'm less sure of that